### PR TITLE
angle-headers: switch to git

### PR DESCRIPTION
**Pull request by zhongfly**

### CODE REVIEW

**Comment by hooke007** [comment]
It's too deprecated.

See https://github.com/google/angle

**Comment by shinchiro** [comment]
I guess gitlab no longer provide free mirroring

**Comment by zhongfly** [comment]
have changed to https://github.com/google/angle

File: packages/angle-headers.cmake [hunk 1]
```diff
@@ -1,11 +1,18 @@
 ExternalProject_Add(angle-headers
-    URL https://gitlab.com/shinchiro/angle/-/archive/main/angle-main.zip?path=include
+    GIT_REPOSITORY https://github.com/google/angle.git
+    SOURCE_DIR ${SOURCE_LOCATION}
+    GIT_CLONE_FLAGS "--sparse --filter=tree:0"
+    GIT_REMOTE_NAME origin
+    GIT_TAG main
+    GIT_CLONE_POST_COMMAND "sparse-checkout set include/EGL include/KHR"
```
Comment (shinchiro) [inline packages/angle-headers.cmake:7]:
```suggestion
    GIT_CLONE_FLAGS "--sparse --filter=tree:0"
    GIT_CLONE_POST_COMMAND "sparse-checkout set include/EGL include/KHR"
```
Move GIT_CLONE_FLAGS to just above GIT_CLONE_POST_COMMAND

**Comment by shinchiro** [comment]
I dont test angle but hope it's still working with git

**Pull request merged**